spec/features/job_seekers_can_filter_vacancies_spec.rb

```diff
@@ -30,28 +30,48 @@
         page.find('.govuk-button[type=submit]').click
       end
 
-      expect(page).to have_content(enfield_vacancy.job_title)
-      expect(page).not_to have_content(penzance_vacancy.job_title)
+      expect(page).to have_content(camden_vacancy.job_title)
+      expect(page).not_to have_content(victoria_vacancy.job_title)
```

Comment:
this change is wrong

Reply:
Thanks for pointing these out. ☝️ 
Fixed in 38a44c4